docs/CONTRIBUTING.md

```diff
@@ -2,10 +2,22 @@
 
 This section generally defines how you can make code contribution. Please follow the below instructions to make code contributions.
 
-Code, PR, commit message format, etc convension I follow when I code. Please follow the links below to get details.
+Code, PR, commit message format, etc. conventions I follow when I code. Please follow the links below to get details.
 
-- [Git branch naming convension](BRANCH-NAMING.md)
+- [Git branch naming convention](BRANCH-NAMING.md)
 
 - [Conventional Commits](https://www.conventionalcommits.org/) for commit messages, and [Commit message format](https://gist.github.com/digitaljhelms/3761873) for what to write
 
 - [Linking a PR to an issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue)
+
+### Examples
+
+#### Branch Names: `{branch_type}/{issue-tracker-id-}issue-one-liner`
+1. feature/2234-infinite-scroll
+2. documentation/3344-linux-installation
+3. test/2222-unit-tests
+
+#### Commits
+1. [Closes #1234] Submit button added
+2. [Fixes #1232] Infinite scroll fixed
+3. Add unit tests for xyz feature
```

Comment:
```suggestion
3. [#333] test: Add unit tests for xyz feature
```

Reply:
Yes. I'll do it.
Should I make a new PR or this one can be used?

Reply:
make a new pr please

Reply:
Okay